pkg/asset/installconfig/clusterid.go

```diff
@@ -23,8 +25,11 @@ type ClusterID struct {
 	UUID string
 
 	// InfraID is an identifier for the cluster that is more human friendly.
-	// This does not have
 	InfraID string
+
+	// LoadBalancerUID is an identifier used for certain platforms to identify resources
```

Comment:
it's important to provide information why infraID cannot be used.

Reply:
The main reason is this one name here:
https://github.com/openshift/kubernetes/blob/a45281f7de40f996f67d0ee7b886add59e7b5e8d/pkg/cloudprovider/providers/gce/gce_loadbalancer_naming.go#L52-L60
Using the infraID could make it exceed the length limit of 63 chars. 
I'll add a comment to code.